engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -114,9 +115,14 @@ object WorkflowActor {
     */
   case class WorkflowActorData(currentLifecycleStateActor: Option[ActorRef],
                                workflowDescriptor: Option[EngineWorkflowDescriptor],
+                               initializationData: AllBackendInitializationData,
                                lastStateReached: StateCheckpoint)
   object WorkflowActorData {
-    def empty = WorkflowActorData(currentLifecycleStateActor = None, workflowDescriptor = None, StateCheckpoint(WorkflowUnstartedState))
+    def empty = WorkflowActorData(
+      currentLifecycleStateActor = None,
+      workflowDescriptor = None,
+      initializationData = AllBackendInitializationData.empty,
+      StateCheckpoint(WorkflowUnstartedState))
```

Comment:
Technically @mcovarr did insert a named parameter before the unnamed parameter so now it *is* his code. Bwahahaha.

@mcovarr Could you make that not be the case plz? :)